packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -342,10 +342,10 @@ $expand-less-icon: map-merge(
       display: inline;
       padding-left: units(0.5);
 
-      & + .usa-nav__secondary-item::before {
-        color: color("base-lighter");
-        content: "|";
-        padding-right: units(0.5);
+      & + .usa-nav__secondary-item {
+        border-left: 1px solid color("base-lighter");
+        margin-left: px-to-rem(5px);
```

Comment:
@mejiaj 
I am curious about why the value is `5px` here instead of matching the `padding-left` value of `units(1)`. I did a little digging, and when I set the `margin-left` to `units(1)`, I saw that the horizontal alignment between the elements was a bit off. Is this the reason for the `5px`? If so, the misalignment seemed to resolve itself when I replaced `float:right` with `display:flex` on `.usa-nav__secondary-links`. Doing so also had the added benefit of making the border height a little smaller. What do you think? Am I making too many assumptions here?